src/styles/variables/_typography.scss

```diff
@@ -0,0 +1,59 @@
+@import url('https://fonts.googleapis.com/css?family=Montserrat:400,700&display=swap');
+
+$main-font: "Montserrat", "Arial", "sans-serif";
+
+body {
+    font-family: $main-font;
+}
+
+h1 {
+    color: $black;
+    font-size: 30px;
+    font-weight: bold;
+    text-transform: lowercase;
+}
+
+h2 {
+    font-size: 16px;
+    font-weight: 700;
+}
+
+h3 {
+    color: $white;
+    font-size: 14px;
+    font-weight: bold;
+    text-transform: lowercase;
+}
+
+p {
+    font-size: 14px;
+}
+
+ul {
+    color: $white;
+    font-size: 12px;
```

Comment:
Deze font-size kan je in een variablen neerzetten. Zeker als waardes meerdere keren worden gebruikt mag dit. Je zou ook voor je heading kunnen verwijzen naar $h1-font-size als voorbeeld op deze manier heb je in een settings files alle "vaste" waardes staan